app/models/Utr.scala

```diff
@@ -33,7 +33,7 @@ object Utr extends (String => Utr) {
   implicit val utrWrite: Writes[Utr] = new SimpleObjectWrites[Utr](_.value)
   implicit val utrRead: Reads[Utr] = new SimpleObjectReads[Utr]("utr", Utr.apply)
 
-  private val validUtrFormat = """^\d{5}\s*?\d{5}"""
+  private val validUtrFormat = """^[0-9]{10}$"""
```

Comment:
I think we should keep the validation here as it was, as it was designed, i.e. to allow with or without a space here.
We can trim out spaces in the transformation when sending to ETMP.
Unless you discussed it with PSR design team and they agreed to change the design for this.

Reply:
We are still allowing the users enter with/without space but we are removing whitespace(s) before executing the validation (line:23). Is this what you mean @kberec ? 

https://design.tax.service.gov.uk/hmrc-design-patterns/unique-taxpayer-reference/
(Remove spaces, characters and extra numbers before validating.)